lib/cobra_commander/component_tree.rb

```diff
@@ -163,7 +163,21 @@ def package_json_path
       end
 
       def combined_deps(json)
-        Hash(json["dependencies"]).merge(Hash(json["devDependencies"]))
+        worskpace_dependencies = build_workspaces(json["workspaces"])
+        dependencies = format_dependencies Hash(json["dependencies"]).merge(Hash(json["devDependencies"]))
+        dependencies.push(worskpace_dependencies).flatten!.uniq!
```

Comment:
Instead of `#push` and `#flatten!`, you could just do `(workspace_dependencies + direct_dependencies).uniq` as the last line.

Reply:
thanks, done that